app/assets/stylesheets/buttons.scss

```diff
@@ -8,14 +8,32 @@
 }
 
 .btn {
+  display: inline-block;
   font-weight: 600;
+  font-size: 14px;
+  padding: 0.75rem 2rem;
+  border-radius: 2rem;
+  text-align: center;
+  border: none;
 
   &-primary {
     background-color: $color-primary-0;
     color: white;
 
     &:hover, &:focus {
       background-color: $color-primary-1;
+      color: white;
+    }
+
+    &-border {
+      background-color: white;
+      box-shadow: inset 0 0 0 2px $color-primary-0;
```

Comment:
Why is this a box-shadow and not a border?

Reply:
When using a border not a box shadow it causes the left button to become bigger than the right. Even when using an inset border because of how box sizing is calculated. We could get around this by giving every button a border but I don't see it as necessary.

Reply:
that makes sense. The way I've always done it in the past is to give the solid buttons a matching colour border as well. Personally, I think that makes more sense than the box shadow, which just looks like a border.